src/main/java/com/khomishchak/cryproportfolio/controllers/GoalsController.java

```diff
@@ -0,0 +1,51 @@
+package com.khomishchak.cryproportfolio.controllers;
+
+import com.khomishchak.cryproportfolio.model.goals.CryptoGoalsRecordUpdateReq;
+import com.khomishchak.cryproportfolio.model.goals.CryptoGoalsTable;
+import com.khomishchak.cryproportfolio.services.GoalsService;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/api/v1/goals")
+public class GoalsController {
+
+    private final GoalsService goalsService;
+
+    public GoalsController(GoalsService goalsService) {
+        this.goalsService = goalsService;
+    }
+
+    @PostMapping("/crypto-table/{accountId}/create")
```

Comment:
Don't use any verbs for the specifying the endpoint path such as `create`
```suggestion
    @PostMapping("/{accountId}/crypto-tables")
```